web/src/fileview/fileview.js

```diff
@@ -253,8 +335,89 @@ function init(initData) {
         $a.focus();
         window.location = $a.attr('href');
       }
-    }
+    } else if (String.fromCharCode(event.which) == 'N' ||
+               String.fromCharCode(event.which) == 'P' ||
+               String.fromCharCode(event.which) == 'M') {
```

Comment:
Instead of having `M`, could we just have the rule that when someone highlights some text, we automatically highlight the rest of the text without their asking?

Reply:
Good point. I wanted to actually add the second color here, basically when you M, then that memories the color.

Let's maybe do that in the follow up? Seems pretty big diff already?